examples/arm/setup.sh

```diff
@@ -23,9 +23,9 @@ target_toolchain=""
 enable_fvps=1
 enable_vela=1
 enable_model_converter=0   # model-converter tool for VGF output
-enable_vgf_lib=0  # vgf reader - runtime backend dependency 
+enable_vgf_lib=0  # vgf reader - runtime backend dependency
 enable_emulation_layer=0  # Vulkan layer driver - emulates Vulkan ML extensions
-mlsdk_manifest_url=""
+mlsdk_manifest_url="https://github.com/arm/ai-ml-sdk-manifest.git"
```

Comment:
From the manifest, most of them are on github.com which is awesome but I do see one on mlplatforms.org (the tosa one). FYI in the past we have seen a lot of reliability issues on that domain, keep an eye out of that if we are enabling this on the CI with high freq like pull.yaml for example.

cc @mergennachin